### PR TITLE
Album endpoint

### PR DESCRIPTION
**Pull request by Timgur**

### CODE REVIEW

File: build/imgur.js [hunk 1]
```diff
@@ -18,7 +18,7 @@
     var imgurAPICall = function imgurAPICall(options) {
         ['method', 'apiUrl', 'path', 'body'].forEach(function (option) {
             if (!options[option]) {
-                throw new Error(option + ' must be specified');
+                throw new Error('' + option + ' must be specified');
```
Comment (Polyrhythm) [inline build/imgur.js]: dpes throwing not work unless you coerce the argument to a string?

Comment (Timgur) [inline build/imgur.js]: this is build,...I did not do this

Comment (laposheureux) [inline build/imgur.js]: Something is going on here. Bjorn's PR the other day removed all of those. I had a quick chat with him about looking at how things are built and how practices change over time. The build file in his PR changed all arguments[x] checks from testing undefined to checking the length of the arguments array and removed all these string coercion safety checks...and this adds the latter back?

Comment (Timgur) [inline build/imgur.js]: how about I nuke my node_modules and resubmit build and see if that does it

Comment (laposheureux) [inline build/imgur.js]: tl;dr: what are you two doing differently to build this that is resulting in those changes flipping back and forth?

**Comment by laposheureux** [comment]
:sheep:it

File: build/imgur.js [hunk 2]
```diff
@@ -48,6 +48,16 @@
         }
     });
 
+    var albumEndpoint = endpoint({
```
Comment (Polyrhythm) [inline build/imgur.js]: if this repo does have babel transpiling we should use `let` or `const` instead of `var`

**Comment by Polyrhythm** [comment]
:cat2:

File: build/imgur.js (hunk 2 above)
Comment (Timgur) [inline build/imgur.js]: this is in build is the output after gulp build

**Pull request merged**